sys/mips/mediatek/mtk_clock.c

```diff
@@ -123,6 +123,8 @@ mtk_clock_get_info(device_t dev, int index, struct fdt_clock_info *info)
 	if (index < 0 || index > 31 || info == NULL)
 		return (EINVAL);
 
+	mask = (1u << index);
+
```

Comment:
This is a separate change, but is otherwise correct. Please split it into a separate commit when you resubmit.